src/openscad.cc

```diff
@@ -791,14 +792,22 @@ int gui(vector<string> &inputFiles, const fs::path &original_path, int argc, cha
     app.connect(&app, SIGNAL(lastWindowClosed()), &app, SLOT(releaseQSettingsCached()));
 	app.connect(&app, SIGNAL(lastWindowClosed()), &app, SLOT(quit()));
 	if (Feature::ExperimentalInputDriver.is_enabled()) {
+		Settings::Settings *s = Settings::Settings::inst();
 #ifdef ENABLE_HIDAPI
         InputDriverManager::instance()->registerDriver(new HidApiInputDriver());
 #endif
 #ifdef ENABLE_SPNAV
-        InputDriverManager::instance()->registerDriver(new SpaceNavInputDriver());
+        SpaceNavInputDriver spaceNav = new SpaceNavInputDriver();
```

Comment:
This looks odd; should this be a pointer?
It could be fixed by `auto` though :)

Reply:
thank you, that is a mistake. I currently have not setup my build system to build the spaceNav driver.